tsup.config.ts

```diff
@@ -5,15 +5,16 @@ import { defineConfig } from 'tsup';
 
 export default defineConfig((options: Options) => ({
   ...options,
-  entry: ['src/index.ts'],
-  treeshake: true,
+
+  cjsInterop: true,
+  clean: true,
+  dts: true,
+  entry: ['src/index.ts', 'src/lib/configure-security.ts'],
   format: ['esm', 'cjs'],
-  silent: !options.watch,
   minify: false,
```

Comment:
don't think we need this, it's the default value ([docs](https://tsup.egoist.dev/#minify-output))

```suggestion
```